edx/edx-enterprise
# [ENT-644] Remove account-level & `EnterpriseCourseEnrollment` consent.

**Description:** This removes any remaining traces of account-level consent and consent for courses tracked by the `EnterpriseCourseEnrollment` model.

**JIRA:** [ENT-644](https://openedx.atlassian.net/browse/ENT-644)

**Merge deadline:** N/A

**Testing instructions:**

Since account-level consent and course-level consent through `EnterpriseCourseEnrollment` were both deprecated in favor of `DataSharingConsent`, this should not have any effect on functionality. However, to be safe, it may be recommended to try out our data sharing consent flows to make sure everything still works as intended.

**Merge checklist:**

  **DO NOT** just add dependencies to `requirements/*.txt` files.

**Post merge:**
